jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -247,8 +247,7 @@ def camelCaseify(s):
 c.KubeSpawner.volume_mounts.extend(get_config('singleuser.storage.extraVolumeMounts', []))
 
 # Gives spawned containers access to the API of the hub
-c.JupyterHub.hub_connect_ip = os.environ['HUB_SERVICE_HOST']
-c.JupyterHub.hub_connect_port = int(os.environ['HUB_SERVICE_PORT'])
+c.JupyterHub.hub_connect_url = 'http://hub'
```

Comment:
Seems like it use port 8081 rather than the default of :80

Reply:
fixed